README.md

```diff
@@ -338,6 +338,43 @@ application on your dashboard.
 
     "new-relic": true
 
+#### datadog
+
+_Default: false_
+
+Enable support for [Datadog](https://www.datadoghq.com/), and specifically:
+- _Datadog APM_ with the [Datadog Tracer extension](https://github.com/DataDog/dd-trace-php/)
+- _Datadog Application Security_ with the [Datadog AppSec extension](https://github.com/DataDog/dd-appsec-php)
+
+_Important:_
+- both extensions will be installed using the Datadog AppSec installer, which is still in beta
+- the Datadog agent has to be installed first ([see the documentation](https://doc.scalingo.com/platform/app/datadog))
+
+To enable Datadog support, set the `extra.paas.datadog` property to `true`:
+
+```
+    "datadog": true
+```
+
+This will automatically install and enable the latest available version of the Datadog Tracer. To disable the Tracer
+or install a specific version, use the `DATADOG_TRACER_VERSION` environment variable (`latest` by default).
+
+Example:
+
+```
+DATADOG_TRACER_VERSION=0.71.0
+```
+
+The Datadog AppSec extension will be disabled by default. To install and enable the AppSec extension, set the
```

Comment:
Is there a reason the AppSec extension is not available by default? I never used Datadog product so I'm not really aware of their offer

Reply:
If we also disable DataDog Tracer by default, the impact of having `"datadog": true` in the composer.json would be a no-op. And I find that strange. If Tracer is a free product, I would allow this by default, and keep AppSec disable. After a bit of thought I think I like your reasoning here :)

Reply:
Fair enough :+1: 

I have squashed commits to keep a clean git history.